src/HealthGPS/converter.cpp

```diff
@@ -78,26 +78,22 @@ RelativeRiskLookup StoreConverter::to_relative_risk_lookup(const core::RelativeR
 RelativeRisk create_relative_risk(const RelativeRiskInfo &info) {
     RelativeRisk result;
     for (const auto &item : info.inputs.diseases()) {
-        auto table = info.manager.get_relative_risk_to_disease(info.disease, item);
-        if (!table.empty() && !table.is_default_value) {
+        const auto table = info.manager.get_relative_risk_to_disease(info.disease, item);
+        if (!table.is_default_value) {
```

Comment:
Is the `!table.is_default_value` check still necessary if we aren't doing default tables anymore?

Reply:
Indeed it's not. Let's just return a `std::optional` instead and drop this field